mllib/src/main/scala/org/apache/spark/mllib/clustering/KMeans.scala

```diff
@@ -156,6 +156,20 @@ class KMeans private (
     this
   }
 
+  // Initial cluster centers can be provided as a KMeansModel object rather than using the
+  // random or k-means|| initializationMode
+  private var initialModel: Option[KMeansModel] = None
+
+  /** Set the initial starting point, bypassing the random initialization or k-means||
```

Comment:
Scala style: comment should begin on line after ```/**``` (See other examples of multi-line comments in this file.)